frontend/app/index.js

```diff
@@ -2,13 +2,30 @@ require('bootstrap/dist/css/bootstrap.min.css');
 
 import React from 'react'
 import { render } from 'react-dom'
+import thunkMiddleware from 'redux-thunk'
+import { createStore, applyMiddleware } from 'redux'
+import { apiMiddleware } from 'redux-api-middleware'
+import { Provider } from 'react-redux'
 import App from './app'
+import randomCat from './reducers/catReducer'
+import { fetchRandomCat } from './actions/catActions'
 
 
+const store = createStore(
+  randomCat,
+  applyMiddleware(
+    apiMiddleware,
+    thunkMiddleware
+  )
+)
+
+store.dispatch(fetchRandomCat())
+  .then(() => console.log(store.getState()))
+
```

Comment:
Actions will be dispatched by containers when the data is needed. Just leaving this here for now so you can check the console and see what the state in the store looks like since there is no UI receiving it yet.